src/game/mario_actions_stationary.c

```diff
@@ -1077,18 +1077,18 @@ s32 check_common_stationary_cancels(struct MarioState *m) {
             return drop_and_set_mario_action(m, ACT_STANDING_DEATH, 0);
         }
     }
-    return FALSE;
+    return ACTION_FINISH;
 }
 
 s32 mario_execute_stationary_action(struct MarioState *m) {
     s32 cancel;
 
     if (check_common_stationary_cancels(m)) {
-        return TRUE;
+        return ACTION_CONTINUE;
     }
 
     if (mario_update_quicksand(m, 0.5f)) {
-        return TRUE;
+        return ACTION_FINISH;
```

Comment:
Got this one wrong

Reply:
EEEEVIL someone what the FUUUCK. i almost approved that...

Reply:
Wdym almost? :trollface:

Reply:
i think we can all agree this is fazana's fault

Reply:
yeah that's true!

Reply:
addressed